src/overlays/actors/ovl_En_Test3/z_en_test3.h

```diff
@@ -7,11 +7,35 @@ struct EnTest3;
 
 typedef void (*EnTest3ActionFunc)(struct EnTest3*, PlayState*);
 
+#define KAFEI_GET_PARAM_1F(thisx) ((thisx)->params & 0x1F)
+#define KAFEI_GET_PARAM_1E0(thisx) (((thisx)->params >> 5) & 0xF)
+
+typedef struct {
+    /* 0x0 */ u8 unk_0;
+    /* 0x1 */ u8 unk_1;
+    /* 0x2 */ u16 textId;
+} struct_D78; // size = 0x4
```

Comment:
```suggestion
} EnTest3_StructD78; // size = 0x4
```
It should at least have the `EnTest3` prefix so it doesn't clash with a different undocumented struct